cfx/README.md

```diff
@@ -34,14 +34,10 @@ To get this extension working for you and your domains, you'll need to fork it a
 `./lib/main.js`:
 
 ```js
-var allowedUris = [
-  'https://respoke.github.io/*',
-  '*.ngrok.com'
-];
-
 var allowedDomains = [
-  'respoke.github.io',
-  '*.ngrok.com'
```

Comment:
we will want to leave ngrok.com, afaik the older version of the ngrok binary still uses ngrok.com.